README.md

```diff
@@ -46,12 +46,28 @@ git clone \
   && cd cardano-graphql
 ```
 ### Build and Run via Docker Compose
-Builds `@cardano-graphql/server` and starts it along with `cardano-ogmios-node`, `cardano-db-sync-extended`, `postgresql`, and `hasura`:
 
 ``` console
-DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose up -d --build && docker-compose logs -f
+./scripts/up.sh --use-cache
```

Comment:
It's a bit sad we loose the docker-only alternative for a bash script. I'd maybe, in the README at least, keep both alternative. Something like:

---

To build with docker and cache layer optimization enabled, run:

```console
$ DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose up -d --build && docker-compose logs -f
```

or alternatively, use the following script at your disposal:

```console
$ ./scripts/up.sh --use-cache
``` 

?

Reply:
I did consider this, but trying to avoid the duplication. Is it not obvious that the script can be inspected to see the commands being issued?

Reply:
Also, the command does get more complex when starting a non-mainnet instance since we need to set the _project name_ too. That's the point I decided it was worth a script to tuck this away, but anyone wanting to dig in can.